src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -74,9 +74,9 @@ const HealthCardDisplay = () => {
       if (!cvxCodes[coding.code]) {
         return coding.system ? `${coding.system}#${coding.code}` : coding.code;
       }
-      return cvxCodes[coding.code];
+      return cvxCodes[coding.code].replace(/&amp;/g, '&');
```

Comment:
Would move this into the iisstandards.js file where the XML handling happens. Also would consider using a replacer function like https://stackoverflow.com/questions/7918868/how-to-escape-xml-entities-in-javascript to handle all the cases (not just &).